Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -199,7 +200,7 @@ private let routers: [Router<Route>] = [
     <¢> post %> lit("invites") %> formField("email", Optional.iso.some >>> opt(.rawRepresentable)) <% end,
 
   .invite <<< .show
-    <¢> get %> lit("invites") %> pathParam(.rawRepresentable >>> .rawRepresentable) <% end,
+    <¢> get %> lit("invites") %> pathParam(.uuid >>> .rawRepresentable) <% end,
```

Comment:
I guess this could also be `.uuid >>> .tagged` which might read a bit nicer.

```suggestion
    <¢> get %> lit("invites") %> pathParam(.uuid >>> .tagged) <% end,
```